course/templates/course/course-base.html

```diff
@@ -48,18 +48,23 @@
     </li>
   {% endif %}
 
-  {% if pperm.view_gradebook %}
+  {% if pperm.view_gradebook or pperm.edit_participation %}
   <li class="dropdown">
     <a href="#" class="dropdown-toggle" data-toggle="dropdown">{% trans "Grading" context "menu item" %}<b class="caret"></b></a>
     <ul class="dropdown-menu">
+        <li role="presentation" class="dropdown-header">{% trans "Participations" %}</li>
+        <li><a href="{% url "relate-view_participant_list" course.identifier %}">{% trans "List of participants" %}</a></li>
+        {% if pperm.edit_participation %}
```

Comment:
I think these should be additional permissions (to edit roles, and to edit tags)

Reply:
Sure.

Reply:
Done.